src/components/admin/SpaceCard.tsx

```diff
@@ -0,0 +1,138 @@
+import { Link } from "react-router-dom";
+import ShowMoreText from "react-show-more-text";
+import { faExternalLinkAlt, faUsers } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import {
+  ADMIN_IA_SPACE_EDIT_PARAM_URL,
+  ATTENDEE_INSIDE_URL,
+  DEFAULT_SHOW_MORE_SETTINGS,
+  DEFAULT_VENUE_LOGO,
+  SPACE_INFO_MAP,
+} from "settings";
+
+import { WorldWithId } from "types/id";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { generateUrl } from "utils/url";
+
+import { useWorldEvents } from "hooks/events";
+
+import { LinkButton } from "./LinkButton";
+
+const cardHeaderStyles = {
+  background:
+    "url(https://images.unsplash.com/photo-1517369611053-aecb0ca6971e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80)",
```

Comment:
If this link is something that we want to keep in the future, it should be moved into settings

Reply:
moved: https://github.com/sparkletown/sparkle/pull/2837/commits/8f6e1c2bd668cc272e0875e7f94ee7f6795ad3d2